docs/security.html

```diff
@@ -246,6 +248,35 @@ <h3><a id="security_sasl" href="#security_sasl">7.3 Authentication using SASL</a
             <tt>zookeeper.sasl.client.username</tt> to the appropriate name
             (<i>e.g.</i>, <tt>-Dzookeeper.sasl.client.username=zk</tt>).</p></li>
 
+            <p>Brokers may also configure JAAS using the broker configuration property <code>sasl.jaas.config</code>.
+            The property name must be prefixed with the listener prefix including the SASL mechanism,
+            i.e. <code>listener.name.{listenerName}.{saslMechanism}.sasl.jaas.config</code>. Only one
+            login module may be specified in the config value. If multiple mechanisms are configured on a
+            listener, configs must be provided for each mechanism using the listener and mechanism prefix.
+            For example,
+                    <pre class="brush: text;">
+        listener.name.sasl_ssl.scram-sha-256.sasl.jaas.config= org.apache.kafka.common.security.scram.ScramLoginModule required \
+            username="admin" \
+            password="admin-secret";
+        listener.name.sasl_ssl.plain.sasl.jaas.config= org.apache.kafka.common.security.plain.PlainLoginModule required \
+            username="admin" \
+            password="admin-secret" \
+            user_admin="admin-secret" \
+            user_alice="alice-secret";</pre>
+
+            If JAAS configuration is defined at different levels, the order of precedence used is:
+            <ul>
+              <li><code>listener.name.{listenerName}.{saslMechanism}.sasl.jaas.config</code></li>
+              <li><code>sasl.jaas.config</code></li>
```

Comment:
Above we mentioned "The property name must be prefixed with the listener prefix including the SASL mechanism".  If we give "sasl.jaas.config" alone, what is the behaviour?

I am getting error when I start broker with "sasl.jaas.config" config property .

Reply:
@omkreddy Thanks for the review. `sasl.jaas.config` should be prefixed with listener name and mechanism for brokers. It shouldn't have been included here, updated.